source/endpoints/plugins/RequiredInfo.py

```diff
@@ -5,35 +5,35 @@
 
 
 @namespace.route("/")
-class Organizations(Resource):
-    """ Show all the organizations that have plugins """
+class Collections(Resource):
+    """ Show all the collections that have plugins """
 
-    @namespace.doc("list_organizations")
+    @namespace.doc("list_collections")
     def get(self):
-        """ Fetch all the organizations that have plugins installed """
+        """ Fetch all the collections that have plugins installed """
         return plugin_logic.get_organizations()
```

Comment:
Still says organization but we can tackle this when we refactor the logic module